CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
   network traffic caused by Hare.
 * [#6099](https://github.com/spacemeshos/go-spacemesh/pull/6099) Adds new metrics to the API to provide insights into
   the performance and behavior of the node's APIs.
+* [#6112](https://github.com/spacemeshos/go-spacemesh/pull/6112) Adds vesting, vault, and drain vault contents to the
+  v2alpha2 Transaction API. Fixes the 'unspecified' transaction type.
```

Comment:
Technically it's a feature, not an improvement. Wdyt?

Reply:
moved to features